drivers/VDI.py

```diff
@@ -858,13 +858,13 @@ def _create_cbt_log(self):
 
         return logpath
 
-    def _activate_cbt_log(self, logname):
+    def _activate_cbt_log(self, logname) -> bool:
```

Comment:
We don't, currently at least, have typing enabled so this is going to have much affect but should be innocuous.